src/main/java/org/imsglobal/caliper/events/ResourceManagementEvent.java

```diff
@@ -25,32 +25,31 @@
 import org.slf4j.LoggerFactory;
 
 @SupportedActions({
-        Action.TAKE_LAUNCHED,
-        Action.TAKE_COMPLETED,
-        Action.SAVED,
-        Action.DELETED
+        Action.ALIASED,
+        Action.DELETED,
+        Action.SAVED
 })
-public class ContentEvent extends Event {
+public class ResourceManagementEvent extends Event {
 
     @JsonIgnore
-    private static final Logger log = LoggerFactory.getLogger(ContentEvent.class);
+    private static final Logger log = LoggerFactory.getLogger(ResourceManagementEvent.class);
 
     /**
-     * Utilize builder to construct ContentEvent
+     * Utilize builder to construct PathEvent
```

Comment:
is this comment correct?

Reply:
No this is incorrect, it should be ResourceManagementEvent.